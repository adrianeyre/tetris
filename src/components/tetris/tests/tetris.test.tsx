import React from 'react';
import { shallow } from 'enzyme';

import Tetris from '../tetris';
import ITetrisProps from '../interfaces/tetris-props';

describe('Tetris', () => {
	it('Should render correctly', () => {
		const defaultProps: ITetrisProps = {};
		const tetris = shallow(<Tetris {...defaultProps} />);
		expect(tetris).toMatchSnapshot();
	});
});