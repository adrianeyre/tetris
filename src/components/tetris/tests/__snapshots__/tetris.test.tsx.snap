// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tetris Should render correctly 1`] = `ShallowWrapper {}`;
